docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -296,6 +297,52 @@ To enable denormals optimization in the application, the `denormals_optimization
 
 @endsphinxdirective
 
+### Sparse weights decompression
+`Sparse weights` are weights where most of the elements are zero. The ratio of the number of zero elements to the number of all elements is called `sparse rate`. Thus, we assume that `sparse weights` are weights with a high sparse rate. In case of `sparse weights`, we can store only non-zero values in memory using special storage structures, which allows us to use memory more efficiently. In turn, this can give us better performance in the high memory bound workloads (e.g., throughput scenario).
+
+`Sparse weights decompression feature` allows to pack weights for Matrix Multiplication operations directly in the CPU plugin at the model compilation stage and store non-zero values in a special packed format. Then, during the execution of the model, the weights are unpacked and used in the computational kernel.
```

Comment:
We need to add smt like: "since weights are loaded from DDR/L3 cache in packed format this signifcantly decreases memory consumpiton and as a a consicuence improve inference performace"

Reply:
added, thanks!